src/components/manager/projects/ProjectJsonSchemaModal.js

```diff
@@ -67,6 +71,7 @@ ProjectJsonSchemaModal.propTypes = {
 
     onOk: PropTypes.func,
     onCancel: PropTypes.func,
+    extraPropertiesSchemaTypes: PropTypes.object.isRequired,
```

Comment:
I think this is not needed since `extraPropertiesSchemaTypes` is obtained from redux with `useSelector`